fixed upload from URL

Bad3r opened the pull request:
- Fixed uploading from URL
- Allowed TravisCI to build all branches

sxo6099 committed:
Updating Python for Dynamic Display

Added test list to try and see if I can get jinja to work.

sxo6099 committed:
Added Jinja Component

Added a display element using Jinja to (hopefully) display elements given by flask.

sxo6099 committed:
Added CSS for other changes

CSS for the container that will eventually have enough in it to be scroll-able; if it were overflowing, could do now.

sxo6099 committed:
Correcting HTML that was Unnecessary

Don't need the video player.

Bad3r committed:
fixed uploading from url

Bad3r committed:
testing travis jobs

Bad3r committed:
testing travis jobs

Bad3r committed:
testing req

Bad3r committed:
allowed all branches to built by TravicCI

brendyFlannel reviewed (approved):
Lookin fly my guy =O

sxo6099 committed:
Added Querying of Database for List

Queries database to grab video names, urls, and uploader.

sxo6099 committed:
Testing to understand formatting of SQL requests

sxo6099 committed:
Splits tuples of SQL requests

Bad3r committed:
fixed url for files

sxo6099 committed:
Converts tuple Components to Str

sxo6099 committed:
Added More Jinja

sxo6099 committed:
Added more Jinja Tests

Bad3r committed:
fixed missing ')'

sxo6099 committed:
Prepare for DB Change

userID will be removed from table.

Z5T1 committed:
Added Video class and updated database

Removed the redundant username field from the videos table. Use a `JOIN
accounts ON videos.userID = accounts.userID` if you need to get the username.

sxo6099 committed:
Merge pull request #80 from CSEC380-Group-12/scott

Pulling in Video Class information

Bad3r committed:
added a function to query the database

sxo6099 committed:
Merge branch 'Main-Page-VideoList' into bad3r

sxo6099 committed:
Merge pull request #81 from CSEC380-Group-12/bad3r

Added a function to query the database

sxo6099 committed:
Made Changes for Different SQL Access Method

Modified front page listing to use new database access function.

Z5T1 committed:
Cleanup & merge Bader's branch

blm4271 committed:
SHANE It prints Jinja

Th acounts table is being acces to test if Jinja can be used to dynaically render the videos.

Z5T1 committed:
Got get_username_from_uid working

blm4271 committed:
SHANE Fetching All from Accounts

Doing so in preparation to swap back to actually fetching from the videos table.

sxo6099 committed:
Changes to select All from Accounts

sxo6099 committed:
Incorrect Comment Fixed

sxo6099 committed:
Fixed another Comment

sxo6099 committed:
Another wrong comment

blm4271 committed:
Merge branch 'Main-Page-VideoList' of https://github.com/CSEC380-Group-12/CSEC380-Final-Project into Main-Page-VideoList

Z5T1 committed:
Merge branch 'Main-Page-VideoList' into scott

Conflicts:
	src/webapp/main.py

sxo6099 committed:
Merge pull request #82 from CSEC380-Group-12/scott

To avoid future conflicts

Z5T1 committed:
Removed debugging print

Bad3r committed:
Merge pull request #84 from CSEC380-Group-12/scott

Scott

Bad3r committed:
fixed adding a video to database

sxo6099 committed:
Merge pull request #85 from CSEC380-Group-12/bad3r

Continued Merging to Stay Up-to-Date

Z5T1 committed:
Got video list working

sxo6099 committed:
Merge branch 'Main-Page-VideoList' into scott

sxo6099 committed:
Merge pull request #86 from CSEC380-Group-12/scott

Correction for SELECT from VIDEOS

Z5T1 committed:
Added get_video_from_id function

Z5T1 committed:
Merge branch 'scott' of github.com:CSEC380-Group-12/CSEC380-Final-Project into scott

blm4271 committed:
Added video player

Added videoplayer and CSS file.

Z5T1 committed:
Merge branch 'Main-Page-VideoList' into scott

Bad3r committed:
sanatized input for sql

Bad3r committed:
Merge pull request #87 from CSEC380-Group-12/scott

Scott

Z5T1 committed:
Got video playback working

sxo6099 committed:
Changing how we Delete

Instead of a delete page, each video page will have a delete button.

sxo6099 committed:
Changing how we delete

sxo6099 committed:
Changing how we delete

Z5T1 committed:
Merge branch 'bad3r' into scott

sxo6099 committed:
Changing how we Delete

No longer going to be a page to navigate to.

sxo6099 committed:
Added delete button to video page

Z5T1 committed:
Fixed get_username_from_uid

Bad3r committed:
working on delete file

Bad3r committed:
Merge pull request #88 from CSEC380-Group-12/scott

Scott

sxo6099 committed:
Merge pull request #89 from CSEC380-Group-12/scott

Update to Play Video

sxo6099 committed:
Delete Button calls /delete and Sumbits vidID

Z5T1 committed:
Got delete working on the backend

Z5T1 committed:
Merge branch 'New-Delete-Method' of github.com:CSEC380-Group-12/CSEC380-Final-Project into New-Delete-Method

Z5T1 committed:
Got delete working on the frontend

Z5T1 committed:
Fixed flask and root users

Z5T1 committed:
Removed debugging prints

Bad3r committed:
Merge pull request #91 from CSEC380-Group-12/scott

Scott
